src/core/View.js

```diff
@@ -586,29 +586,36 @@ export default class View extends Component {
           )
         ),
       ];
-      return this.selections
-        .map((v) => {
-          const { prop, compositeID, displayPosition } = v.getProperties();
-
-          // Return false to mark this item for removal
-          if (prop == null) return false;
-
-          return {
-            worldPosition: Array.from(
-              this.openglRenderWindow.displayToWorld(
-                displayPosition[0],
-                displayPosition[1],
-                displayPosition[2],
-                this.renderer
-              )
-            ),
-            displayPosition,
-            compositeID, // Not yet useful unless GlyphRepresentation
-            ...prop.get('representationId'),
-            ray,
-          };
-        })
-        .filter(Boolean);
+      return (
+        this.selections
+          // sort by most dominant objects first.
+          .sort(
```

Comment:
I simply added the .sort predicate. Other changes are perhaps due to `npm run fix:prettier`.